.yarnrc.yml

```diff
@@ -0,0 +1,11 @@
+nodeLinker: node-modules
+
+packageExtensions:
+  stylelint-config-recommended-scss@*:
+    peerDependencies:
+      postcss: 8
+  stylelint-config-standard-scss@*:
+    peerDependencies:
+      postcss: 8
```

Comment:
If this was copied from PIE, I _think_ you can probably remove the stylelint-* stuff. It was a very old bug when the repo was first created, and I don't think it's present now, but the code just hasn't been removed yet 👍🏼

Reply:
oh amazing, thank you! I'll remove and test 👍